dev/remsh.tls

```diff
@@ -0,0 +1,30 @@
+#!/bin/bash
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not
+# use this file except in compliance with the License. You may obtain a copy of
+# the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+# WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+# License for the specific language governing permissions and limitations under
+# the License.
+
+if [ -z $NODE ]; then
+    if [ -z $1 ]; then
+        NODE=1
+    else
+        NODE=$1
+    fi
+fi
+
+if [ -z $HOST ]; then
+    HOST="127.0.0.1"
+fi
+
+NAME="remsh$$@$HOST"
+NODE="node$NODE@$HOST"
+COOKIE=monster
+rootdir="$(cd "${0%/*}" 2>/dev/null; echo "$PWD")"
+erl -name $NAME -remsh $NODE -setcookie $COOKIE -hidden -proto_dist inet_tls -ssl_dist_optfile "${rootdir}/ssl_dist.conf"
```

Comment:
Could it be possible to do something like NODE here for ssl_dist file and have the users pass it as an option. The idea is that we'd have remsh.tls work in a release not just with `./dev/run ...`. In case of a release they might have put the file in `$release_root/etc/couch_ssl_dist.conf` for example